apps/docs/pages/guides/auth/managing-user-data.mdx

```diff
@@ -87,6 +87,36 @@ If you need to fetch a full list of user profiles, we supply a `service_key` whi
 
 Make sure you _NEVER_ expose this publicly. But it can be used on the server-side to fetch all of the profiles.
 
+## Accessing User Metadata
+
+Users can pass in metadata on sign up. User metadata is stored on the `raw_user_meta_data` column of the `auth.users` table.
+
+Here's an example of how to add metadata fields on signup:
+
+```
+const { data, error } = await supabase.auth.signUp(
+  {
+    email: 'example@email.com',
+    password: 'example-password',
+    options: {
+      data: {
+        first_name: 'John',
+        age: 27,
+      }
+    }
+  }
+)
+```
+
+To view the metadata:
+
+```js
+const {
+  data: { user },
+} = await supabase.auth.getUser()
+let metadata = user.user_metadata
+```
```

Comment:
Consider adding tabs here, one with:

select raw_user_meta_data
from auth.users;

Reply:
Will add this in the FLUP with section about admin updates